src/service/stream.rs

```diff
@@ -69,7 +69,9 @@ impl ReadServiceRequest {
         req: types::ReadRequest,
     ) -> Self {
         Self {
-            client,
+            client: client
+                .accept_compressed(CompressionEncoding::Zstd)
+                .accept_compressed(CompressionEncoding::Gzip),
```

Comment:
Any reason to accept gzip since we know service will support zstd?

Reply:
Not really, just because we can. Works as a fallback.

Reply:
Removed Gzip